lib/public_activity/orm/active_record/activity.rb

```diff
@@ -16,7 +16,7 @@ class Activity < ::ActiveRecord::Base
           belongs_to :recipient, :polymorphic => true
         end
         # Serialize parameters Hash
-        serialize :parameters, Hash
+        serialize :parameters, Hash unless [:json, :jsonb, :hstore].include?(attribute_types["parameters"].type)
```

Comment:
You should probably use `columns_hash['parameters'].type` as `attribute_types` doesn't seem to work with Rails 4.